oracle/README.md

```diff
@@ -14,6 +14,8 @@ Get metrics from Oracle Database servers in real time to visualize and monitor a
 
 To use the Oracle integration, either install the Oracle Instant Client libraries, or download the Oracle JDBC Driver. Due to licensing restrictions, these libraries are not included in the Datadog Agent, but can be downloaded directly from Oracle.
 
+Note: JPype, one of the library used by the agent, depends specifically on the [Microsoft Visual C++ Runtime 2015][13]. Make sure this runtime is installed on your system.
```

Comment:
```suggestion
**Note**: JPype, the library used by the Agent if JDBC is chosen, depends specifically on the [Microsoft Visual C++ Runtime 2015][13]. Make sure this runtime is installed on your system.
```

Reply:
I didn't want to say that this is a problem only if JDBC is chose. Yes there is this PR https://github.com/DataDog/integrations-core/pull/6118 but this is not merged yet. So the C++ runtime is a requirement no matter what.